.travis.yml

```diff
@@ -29,11 +29,11 @@ after_success:
     npm run coverage:upload;
   fi
   if [[ $TRAVIS_BRANCH == 'release' && $TRAVIS_PULL_REQUEST == 'false' ]]; then
```

Comment:
Yes, seems like a good idea. We then need a new action in repo-cooker to get name of current branch.